tests/qos/test_buffer_traditional.py

```diff
@@ -0,0 +1,222 @@
+import logging
+
+import pytest
+
+from tests.common.utilities import wait_until
+from tests.common.helpers.assertions import pytest_assert
+
+def load_lossless_info_from_pg_profile_lookup(duthost):
```

Comment:
please make this an autouse module scoped pytest fixture

Reply:
Fixed